### DISCUSSION TITLE
TSMuxer import or EL issue?

### DISCUSSION THREAD

**Discussion by quamt** [quietvoid/dovi_tool#237] [Q&A]
Hello,

I am not sure if I do something wrong but I get a error message when I try to import the extracted EL.hevc into TSmuxer.

I try converting from DoVi 5 to 7 which will create a BL.hevc and an EL.hevc.
When importing the BL all is good but the EL gives an error that it can't read the file.

Any help would be appreciated. 

Thank you.

**Comment by quietvoid** [comment]
Profile 5 does not have an EL.
The file has to be recognized as a video by MediaInfo.
What you have is probably just the RPU.

**Comment by quamt** [comment]
@quietvoid 
I am able to extract an RPU from the layer 5.

But if the Layer 5 does not consist of EL why does it create a EL.hevc?

**Reply by quietvoid** [reply]
Because you're using the `demux` subcommand instead of `extract-rpu`.
`demux` is specifically for profile 7 only, and there is no internal detection of the profile to change the behaviour.

What you ask for is what you get.
